akka-actor-tests/src/test/scala/akka/actor/dungeon/DispatchSpec.scala

```diff
@@ -6,6 +6,7 @@ package akka.actor.dungeon
 
 import akka.actor.Actor
 import akka.actor.Props
+import akka.actor.actorRef2Scala
```

Comment:
will this be needed also for end users when they are using `!` in Scala 3? That would be very bad user experience.

Reply:
Yes, at best of my knowledge, or this specific import or the `import akka.actor._` wildcard.